flowmachine/flowmachine/core/connection.py

```diff
@@ -275,9 +275,6 @@ def _known_dates(self, table, schema):
             if x.isnumeric()
         )
 
-    @cached(
-        TTLCache(1024, 120)
-    )  # Many dates to cache, two minutes seems reasonable to balance db access against speed boost
```

Comment:
Just for my understanding, what's the reason for removing this?

Reply:
Just to record the reasoning - this function is only called by `available_dates`, which is itself cached. The periodic update function added to server.py skips the cache for `available_dates` so it will always hit the db and create any table objects needed. That isn't guaranteed if this method is _also_ cached.